HADiscovery.py

```diff
@@ -59,10 +61,7 @@ def decamelcase(str):
         "component_type": "binary_sensor",
         "payload_values": {"device_class": "door", "payload_on": "Open", "payload_off": "Closed"},
     },
-    539: {  # BSH.Common.Setting.PowerState
-        "component_type": "binary_sensor",
-        "payload_values": {"device_class": "power"},
-    },
+    539: {"component_type": "switch"},  # BSH.Common.Setting.PowerState
```

Comment:
See https://github.com/hcpy2-0/hcpy/issues/103 PowerState should be an enum with On/Off/Standby